draft-ietf-quic-qpack.md

```diff
@@ -769,3 +769,19 @@ those authors is gratefully acknowledged, as well as ideas from:
 * Ian Swett
 
 * Dmitri Tikhonov
+
+
+# Change Log
+
+> **RFC Editor's Note:**  Please remove this section prior to publication of a
+> final version of this document.
+
+## Since draft-ietf-quic-qpack-00
```

Comment:
I think you mean "Since draft-ietf-quic-qcram-00" -- this will be qpack-00, and needs to be submitted as replacing -qcram.